### PR TITLE
Allowed filtering of ended arrangements

### PR DESCRIPTION
**Pull request by jaysengardner**

### CODE REVIEW

File: src/caretogether-pwa/src/Families/FamilyScreen.tsx [hunk 1]
```diff
@@ -123,7 +124,10 @@ export function FamilyScreen() {
 				return true;
 			} else if (opt === ArrangementFilterOptionLabel.Cancelled && arrangement.cancelledAtUtc !== undefined) {
 				return true;
-			}
+			} else if (opt === ArrangementFilterOptionLabel.Ended && arrangement.cancelledAtUtc !== undefined) {
+        return true;
+
+      }
```
Comment (LarsKemmann) [inline src/caretogether-pwa/src/Families/FamilyScreen.tsx:130]: Please make sure that the whitespace/indentation levels are aligned correctly.

File: src/caretogether-pwa/src/Families/FamilyScreen.tsx [hunk 2]
```diff
@@ -123,7 +124,10 @@ export function FamilyScreen() {
 				return true;
 			} else if (opt === ArrangementFilterOptionLabel.Cancelled && arrangement.cancelledAtUtc !== undefined) {
 				return true;
-			}
+			} else if (opt === ArrangementFilterOptionLabel.Ended && arrangement.cancelledAtUtc !== undefined) {
```
Comment (LarsKemmann) [inline src/caretogether-pwa/src/Families/FamilyScreen.tsx:127]:
```suggestion
			} else if (opt === ArrangementFilterOptionLabel.Ended && arrangement.endedAtUtc !== undefined) {
```
I think the original code you wrote has a logic bug in it? Take a look at this proposed edit and see if that's the case. Either way, when you know it's correct, include a screenshot or two in your PR showing the correct logic with some sample data that you've entered to demonstrate that you've tested it.

**Review by LarsKemmann — CHANGES_REQUESTED** [review]
A decent start but needs some more work and testing.

**Comment by jaysengardner** [comment]
Fixed a logic error that was present in my original code. Images provided to show that it is now working as intended.

**Comment by jaysengardner** [comment]
Screenshot (1)

**Comment by jaysengardner** [comment]
Screenshot (2)

**Comment by LarsKemmann** [comment]
> Fixed a logic error that was present in my original code. Images provided to show that it is now working as intended.

I don't see an ended arrangement in your screenshots, nor any indication of what you set the filter options to.

**Review by LarsKemmann — APPROVED** [review]
Looks great, thank you!!

**Pull request merged**